homeassistant/helpers/entity_component.py

```diff
@@ -74,14 +74,11 @@ def add_entities(self, new_entities):
 
                 entity.update_ha_state()
 
-        if self.group is None and self.group_name is not None:
-            self.group = group.Group(self.hass, self.group_name,
-                                     user_defined=False)
-
         if self.group is not None:
             self.group.update_tracked_entity_ids(self.entities.keys())
 
-        self._start_polling()
+        for platform_component in self.entity_platform_components.values():
```

Comment:
Each platform will start polling now instead of only the platform that contained the device that has to be polled.